packages/snaps-webpack-plugin/src/manifest.ts

```diff
@@ -18,14 +17,11 @@ export async function writeManifest(
   data: string,
   writeFileFn: WriteFileFunction = fs.writeFile,
 ) {
-  const config = await resolveConfig(path, {
-    editorconfig: true,
-  });
-
-  const formattedManifest = format(data, {
-    ...config,
+  const formattedManifest = await format(data, {
```

Comment:
Wondering if we should just use Prettier 2 instead since we are forced to keep it around for now anyways?

Reply:
With the ESLint 9 changes we need to bump to Prettier 3 as well. We could keep Prettier 2 around for this specifically, but I don't think it would be able to load Prettier 3 configs 😕

Reply:
> What would the downside be of the CLI still using Prettier 2?

That it won't be able to load Prettier 3 configs 😅 I assume Snaps will want to update to ESLint 9, Prettier 3, etc. as well, so I'm not sure if there's a benefit to keeping Prettier 2 around.